### DISCUSSION TITLE
Default name on Vertipaq Export file

### DISCUSSION THREAD

**Discussion by nariusp** [TabularEditor/TabularEditor3#1062] [Ideas]
Not really a bug but it would be great if the Export (Vertipaq) would supply a default name for the export file as does DAX Studio. It's a real time saver as it takes the name from the model. Right now in TE3 it comes up blank when you Export.

**Comment by mlonsk** [comment]
This is implemented in Tabular Editor 3 a while ago. Appologies for not updating this at the time.
